homeassistant/components/switch/fritzdect.py

```diff
@@ -0,0 +1,151 @@
+"""
+Support for FRITZ!DECT Switches.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.fritzdect/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.switch import (SwitchDevice, PLATFORM_SCHEMA)
+from homeassistant.const import (
+    CONF_HOST, CONF_PASSWORD, CONF_USERNAME)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import TEMP_CELSIUS, STATE_UNKNOWN
+
+REQUIREMENTS = ['fritzhome==1.0.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+# Standard Fritz Box IP
+DEFAULT_HOST = 'fritz.box'
```

Comment:
To my understanding, fritz.box is the universal (IPv4 and IPv6) way of addressing a fritz box. Having a hardcoded IPv4 IP might fail in certain networks.